sr_gui_change_controllers does not loadup in installed env due to missing file

[ERROR] [1448651317.382087278]: Skipping XML Document "/work/catkin_ws_indigo/install/share/sr_gui_change_controllers/sr_change_controllers_plugin.xml" which had no Root Element.  This likely means the XML is malformed or missing.

--- a/sr_gui_change_controllers/CMakeLists.txt
+++ b/sr_gui_change_controllers/CMakeLists.txt
@@ -150,6 +150,10 @@ include_directories(
 #   DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
 # )
 
+install( DIRECTORY uis DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION} )
+
+install( FILES sr_change_controllers_plugin.xml DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION} )
+
 #############
 ## Testing ##
 #############

